packages/host-inventory/doc/classes/Configuration.md

```diff
@@ -31,7 +31,7 @@
 
 #### Defined in
 
-[configuration.ts:67](https://github.com/RedHatInsights/javascript-clients/blob/master/packages/host-inventory/configuration.ts#L67)
+[configuration.ts:67](https://github.com/gkarat/javascript-clients/blob/master/packages/host-inventory/configuration.ts#L67)
```

Comment:
similar changes should not occur. You may need to change your git origin for generating.